### ISSUE TITLE
The blurred image is flipped vertically in Windows(Unity2017.2).

### ISSUE THREAD

**Issue by mob-sakai** [mob-sakai/StaticBluredScreen#1]

**Comment by mob-sakai** [comment]
https://docs.unity3d.com/Manual/SL-PlatformDifferences.html
```c++
#if UNITY_UV_STARTS_AT_TOP
if (_MainTex_TexelSize.y < 0)
        uv.y = 1-uv.y;
#endif
```

**Comment by mob-sakai** [comment]
Fixed in 0.2.0
